build_options.cmake

```diff
@@ -9,10 +9,12 @@
 # http://www.cmake.org/Wiki/CMake_Cross_Compiling
 # https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html
 # http://stackoverflow.com/questions/12802377/in-cmake-how-can-i-find-the-directory-of-an-included-file
-if ( WIN32 )
+if ( WIN32 ) # TODO: add detection of Windows Phone / Windows Universal platform
```

Comment:
Not really - WP uses default MSVC toolchain.

For example, to configure code for WP 8.0 ARM, you should use:

    cmake -G "Visual Studio 11 2012 ARM" -DCMAKE_SYSTEM_NAME=WindowsPhone -DCMAKE_SYSTEM_VERSION=8.0

For WP 8.1, similarly:

    cmake -G "Visual Studio 12 2013 ARM" -DCMAKE_SYSTEM_NAME=WindowsPhone -DCMAKE_SYSTEM_VERSION=8.1

And for Windows Mobile 10.0 / Universal Windows App:

    cmake -G "Visual Studio 14 2015 ARM" -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=10.0

Of course, specifiyng `Win64` instead of `ARM` will generate 64-bit VS project and not specifying anything will generate 32-bit (x86) VS project. On WindowsPhone, `x86` is usually interpreted as simulator build, while WindowsStore can be either `x86`, `ARM` or `Win64` (in Windows 8.0 and 8.1, WindowStore referred to Metro apps which used different MSVCRT than WindowsPhone - on Windows 10, WindowsStore system name refers to universal windows app).